services/management/ia/rules_service.go

```diff
@@ -17,38 +17,225 @@
 package ia
 
 import (
+	"bytes"
 	"context"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"strings"
 
 	"github.com/percona-platform/saas/pkg/common"
 	iav1beta1 "github.com/percona/pmm/api/managementpb/ia"
+	"github.com/percona/promconfig"
 	"github.com/pkg/errors"
 	"github.com/sirupsen/logrus"
 	"google.golang.org/grpc/codes"
 	"google.golang.org/grpc/status"
 	"gopkg.in/reform.v1"
+	"gopkg.in/yaml.v3"
 
 	"github.com/percona/pmm-managed/models"
 	"github.com/percona/pmm-managed/services"
+	"github.com/percona/pmm-managed/utils/dir"
 )
 
+const rulesDir = "/etc/ia/rules"
+
 // RulesService represents API for Integrated Alerting Rules.
 type RulesService struct {
-	db        *reform.DB
-	l         *logrus.Entry
-	templates *TemplatesService
-	vmalert   vmAlertService
+	db           *reform.DB
+	l            *logrus.Entry
+	templates    *TemplatesService
+	vmalert      vmAlert
+	alertManager alertManager
+	rulesPath    string // used for testing
+
 }
 
 // NewRulesService creates an API for Integrated Alerting Rules.
-func NewRulesService(db *reform.DB, templates *TemplatesService, vmalert vmAlertService) *RulesService {
+func NewRulesService(db *reform.DB, templates *TemplatesService, vmalert vmAlert, alertManager alertManager) *RulesService {
+	l := logrus.WithField("component", "management/ia/rules")
+
+	err := dir.CreateDataDir(rulesDir, "pmm", "pmm", dirPerm)
```

Comment:
you might want to add another call to change the owners of its parent dir `"/etc/ia/"` last time I was doing it, the owner:group of the parent dir turned out to be different from the child so I added two calls with separate paths in the templates service

Reply:
Does it caused some problems in you case? And did you find reason why parend dir had differerent owner?

Reply:
> Does it caused some problems in you case?

not that I can remember, for context you can take a look at this thread https://github.com/percona/pmm-managed/pull/558/#discussion_r536101878

> And did you find reason why parend dir had differerent owner?

no I couldn't

Reply:
Well, lets keep it as is until there is no problems. But I will keep in mind that such case is possible.

Reply:
There should not be a problem for `/etc/…` dirs that are internal for container. `/srv/…`, however, can be on an external volume, and there problems could happen